inference/inference_basicvsr.py

```diff
@@ -0,0 +1,94 @@
+import argparse
+import cv2
+import glob
+import numpy as np
+import os
+import shutil
+import torch
+
+from basicsr.archs.basicvsr_arch import BasicVSR
+from basicsr.utils.img_util import img2tensor, tensor2img
+
+
+def cache_imgs(paths, device='cpu'):
+    imgs = []
+    imgnames = []
+    for idx, path in enumerate(paths):
+        imgname = os.path.splitext(os.path.basename(path))[0]
+        print('Testing', idx, imgname)
+        img = cv2.imread(path).astype(np.float32) / 255.
+        imgs.append(img)
+        imgnames.append(imgname)
+
+    imgs = img2tensor(imgs, bgr2rgb=True, float32=True)
+    imgs = torch.stack(imgs, dim=0)
+    imgs = imgs.unsqueeze(0).to(device)
+    return imgs, imgnames
+
+
+def inference(imgs, imgnames, model, args):
+    # inference
+    outputs = model(imgs)
+
+    # save imgs
+    outputs = outputs.squeeze()
+    outputs = list(outputs)
+    for output, imgname in zip(outputs, imgnames):
+        output = tensor2img(output)
+        cv2.imwrite(os.path.join(args.output, f'{imgname}_BasicVSR.png'), output)
+
+
+def main():
+    parser = argparse.ArgumentParser()
+    parser.add_argument(
+        '--model_path',
+        type=str,
+        default=  # noqa: E251
```

Comment:
I think in the new 120 line format, there is no need to use # noqa: E251 and # noqa: E501

Just put them into one line may be OK